cluster.c

```diff
@@ -886,16 +895,16 @@ static void appendClusterNodeString(RedisModuleString *ret, char node_id[41], No
                                                            pong_recv,
                                                            epoch,
                                                            link_state,
-                                                           RedisModule_StringPtrLen(slots, NULL));
+                                                           (int) slots_len, slots_str);
```

Comment:
is the point of this change to protect against bugs in the creation of the slot_str? where the pointer we get out of it isn't null terminated where we expect it to be?

Reply:
Yes, `RedisModuleString` is not formally guaranteed to be null terminated.